policyengine_us/reforms/congress/romney/family_security_act_two_point_o/family_security_act_two_point_o.py

```diff
@@ -0,0 +1,109 @@
+from policyengine_us.model_api import *
+from policyengine_core.periods import period as period_
+
+
+def create_family_security_act_two_point_o() -> Reform:
+    class ctc_phase_in_rate(Variable):
+        value_type = float
+        entity = TaxUnit
+        definition_period = YEAR
+        unit = USD
+        label = "Child Tax Credit phase-in rate"
+
+        def formula(tax_unit, period, parameters):
+            income = tax_unit("adjusted_gross_income", period)
+            p = parameters(
+                period
+            ).gov.contrib.congress.romney.family_security_act_two_point_o.ctc.phase_in
+            return min_(income / p.earnings_limit, 1)
+
+    class ctc(Variable):
+        value_type = float
+        entity = TaxUnit
+        label = "Child Tax Credit"
+        unit = USD
+        documentation = "Total value of the non-refundable and refundable portions of the Child Tax Credit."
+        definition_period = YEAR
+        reference = "https://www.law.cornell.edu/uscode/text/26/24#a"
+
+        def formula(tax_unit, period, parameters):
+            maximum_amount = tax_unit("ctc_maximum_with_arpa_addition", period)
+            phase_in_rate = tax_unit("ctc_phase_in_rate", period)
+            phased_in_max_amont = maximum_amount * phase_in_rate
+            reduction = tax_unit("ctc_phase_out", period)
+            return max_(0, phased_in_max_amont - reduction)
+
+    class ctc_qualifying_children(Variable):
+        value_type = int
+        entity = TaxUnit
+        label = "CTC-qualifying children"
+        documentation = (
+            "Count of children that qualify for the Child Tax Credit"
+        )
+        definition_period = YEAR
+        reference = "https://www.law.cornell.edu/uscode/text/26/24#c"
+
+        def formula(tax_unit, period, parameters):
+            total_children = tax_unit.sum(
+                tax_unit.members("ctc_qualifying_child", period)
+            )
+            p = parameters(
+                period
+            ).gov.contrib.congress.romney.family_security_act_two_point_o.ctc
+            return min_(total_children, p.child_cap)
+
+    class eitc_maximum(Variable):
+        value_type = float
+        entity = TaxUnit
+        label = "Maximum EITC"
+        definition_period = YEAR
+        reference = "https://www.law.cornell.edu/uscode/text/26/32#a"
+        unit = USD
+
+        def formula(tax_unit, period, parameters):
+            child_count = tax_unit("eitc_child_count", period)
+            p = parameters(
+                period
+            ).gov.contrib.congress.romney.family_security_act_two_point_o.eitc.amount
+            filing_status = tax_unit("filing_status", period)
+            joint = filing_status == filing_status.possible_values.JOINT
+            return where(
+                joint, p.joint.calc(child_count), p.single.calc(child_count)
+            )
+
+    class reform(Reform):
```

Comment:
don't we also need to change the list of refundable/non-refundable credits? ctc becomes fully refundable right?

Reply:
I thought we had a switch in the app as the [fully_refundable.yaml](https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/parameters/gov/irs/credits/ctc/refundable/fully_refundable.yaml)